astropy/time/tests/test_basic.py

```diff
@@ -167,14 +167,14 @@ def test_properties(self):
         """Use properties to convert scales and formats.  Note that the UT1 to
         UTC transformation requires a supplementary value (``delta_ut1_utc``)
         that can be obtained by interpolating from a table supplied by IERS.
-        This will be included in the package later."""
+        This is tested separately."""
 
         t = Time('2010-01-01 00:00:00', format='iso', scale='utc')
         t.delta_ut1_utc = 0.3341  # Explicitly set one part of the xform
         assert allclose_jd(t.jd, 2455197.5)
         assert t.iso == '2010-01-01 00:00:00.000'
         assert t.tt.iso == '2010-01-01 00:01:06.184'
-        assert t.tai.iso == '2010-01-01 00:00:34.000'
```

Comment:
Just changed it to `fits` -- so, still test that `tai` works but now also test the `fits` output (which of course is identical to that of `isot` but adds the scale).